public/js/components/pay-method-icon.jsx

```diff
@@ -0,0 +1,37 @@
+import cx from 'classnames';
+import React, { Component, PropTypes } from 'react';
+
+// Just a convenience mapping for payMethods from payMethod-validator
+// to shorted classes used in CSS.
+const payMethodTypeMap = {
+  'american-express': 'amex',
+  'diners-club': 'diners',
+  'master-card': 'mastercard',
+};
```

Comment:
hmm, this is going to need a lot of refactoring to wedge in paypal ;) I guess it's better to at least prepare by calling it `PayMethodIcon`.